source/BrightcoveOS .NET-MAPI-Wrapper/Model/Items/BrightcoveCaptionSource.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Web.Script.Serialization;
+using BrightcoveMapiWrapper.Serialization;
+using BrightcoveMapiWrapper.Util;
+
+namespace BrightcoveMapiWrapper.Model.Items
+{
+    /// <summary>
+    /// A source that provides captions for a video
+    /// </summary>
+    /// <remarks>
+    /// See https://docs.brightcove.com/en/video-cloud/media/references/reference.html#CaptionSource
+    /// </remarks>
+    public class BrightcoveCaptionSource : BrightcoveItem, IJavaScriptConvertable
+    {
+        /// <summary>
+        /// A Boolean indicating whether a CaptionSource is usable.
+        /// </summary>
+        public bool Complete { get; set; }
```

Comment:
Their docs explanation sucks. My understanding is that equals 'is published' or 'not a draft'. Just speaking out loud